.github/workflows/run_functional_product_tests.yaml

```diff
@@ -0,0 +1,43 @@
+# Author: James Garriss
+# Purpose: Run the product functional tests every night
+
+name: Nightly Product Functional Tests
+
+# Run this workflow at midnight 
+# on every Mon to Fri (b/c GMT -5)
+on:
+  schedule:
+  - cron: '0 4 * * 1-5'
+  workflow_dispatch: 
+  push:
+
+jobs:
+  sharepoint-test:
+    name: SharePoint Test
+    runs-on: windows-latest
+    defaults:
+      run:
+        shell: powershell 
+    steps:
+    - name: Checkout Repo
+      uses: actions/checkout@v4
+    - name: Create PFX
+      run: |
+        New-Item -ItemType file -Path ./key.txt
+        Set-Content -Path ./key.txt -Value $env:PfxBase64
+        certutil -decode ./key.txt ./key.pfx
+        dir
+      env:
+        PfxBase64: ${{ secrets.NIGHTLY_TEST_BUILD_PFX }}
+    - name: Import PFX
+      run: Import-PfxCertificate -Password (ConvertTo-SecureString -String $env:PfxPassword -AsPlainText -Force) -CertStoreLocation Cert:\CurrentUser\My -FilePath ./key.pfx
+      env:
+        PfxPassword: ${{ secrets.NIGHTLY_TEST_BUILD_PW }}
+    - name: Setup for Scuba
+      run: ./SetUp.ps1
+    - name: Install Selenium
+      run: Install-Module -Name Selenium -RequiredVersion 3.0.1
```

Comment:
Not bad but why restrict to version 3.0.1 instead of just latest?

Reply:
done